lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarFooter.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+
+import NxGlobalSidebarFooter, { NxGlobalSidebarFooterProps as Props }
+  from '../NxGlobalSidebarFooter';
+
+import NxTextLink from '../../NxTextLink/NxTextLink';
+
+describe('NxGlobalSidebarFooter', function() {
+  const minimalProps: Props = {
+        supportText: 'Support for RSC',
+        supportLink: '#supporturl',
+        releaseText: 'React Shared Components',
+        releaseNumber: '3.1.4',
+        productTagLine: 'Powered by PLAID VILLAIN',
+        showCreatedBy: true
+      },
+      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxGlobalSidebarFooter, minimalProps);
+
+  it('renders an <footer> with nx-global-sidebar__footer class', function() {
+    expect(getShallowComponent()).toMatchSelector('footer.nx-global-sidebar__footer');
+  });
+
+  it('renders the Support div with a link with the passed href and text', function() {
+    const support = getShallowComponent().find('div.nx-global-sidebar__support'),
+        supportLink = getShallowComponent().find('div.nx-global-sidebar__support').find(NxTextLink);
+
+    expect(support).toContainMatchingElement('span.nx-global-sidebar__support-text');
+    expect(support.find('span.nx-global-sidebar__support-text')).toHaveText('Support for RSC');
+    expect(supportLink).toHaveProp('href', '#supporturl');
+  });
+
+  it('renders the Release div with a text and version number', function() {
```

Comment:
This tests when `releaseText` and `releaseNumber` are both present.  You also need 3 more tests for when either, and both, aren't present.  I believe you will find that things do not work correctly when both are absent, due to your usage of square brackets instead of parentheses in the conditional expression around the rendering of `.nx-global-sidebar__release`

Reply:
> This tests when `releaseText` and `releaseNumber` are both present. You also need 3 more tests for when either, and both, aren't present. I believe you will find that things do not work correctly when both are absent, due to your usage of square brackets instead of parentheses in the conditional expression around the rendering of `.nx-global-sidebar__release`

At least part of this is no longer true bc we've dropped `releaseName` altogether: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/a3582fe31ea98556bd1bdbc6dcace8e73f18cc2d

Reply:
> At least part of this is no longer true bc we've dropped releaseName altogether: a3582fe

Is the most typical expected usage that it would just be something like "Release 1.2.3" or is it expected to be something like "React Shared Components 1.2.3" or "React Shared Components Release 1.2.3" as you had before?  If it's expected to be one of the latter I think the prop would be better named `releaseText`, and our examples should reflect that

Reply:
> > At least part of this is no longer true bc we've dropped releaseName altogether: [a3582fe](https://github.com/sonatype/sonatype-react-shared-components/commit/a3582fe31ea98556bd1bdbc6dcace8e73f18cc2d)
> 
> Is the most typical expected usage that it would just be something like "Release 1.2.3" or is it expected to be something like "React Shared Components 1.2.3" or "React Shared Components Release 1.2.3" as you had before? If it's expected to be one of the latter I think the prop would be better named `releaseText`, and our examples should reflect that

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/b0313b5bed7fc1d67f877619fb6cf42bc4a34a69

Reply:
> Sure: b0313b5

> and our examples should reflect that

What I meant here is that at least some of the examples should go back to saying something like "React Shared Components 1.2.3"

Reply:
> > Sure: [b0313b5](https://github.com/sonatype/sonatype-react-shared-components/commit/b0313b5bed7fc1d67f877619fb6cf42bc4a34a69)
> 
> > and our examples should reflect that
> 
> What I meant here is that at least some of the examples should go back to saying something like "React Shared Components 1.2.3"

Some did, but I think that they all do now: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/4916e577a5ea2d0331ef672b9cc87fe76505d37c

Make sure you look at the page for `NxGlobalSidebarFooter`.

Reply:
> Some did, but I think that they all do now: 4916e57
>
> Make sure you look at the page for NxGlobalSidebarFooter.

This didn't address what I said.  It is my understanding that, most typically, the text for this property will be the product name followed by the version number.  In your examples you just have "Release" followed by the version number.  I am suggesting that we make them more realistic by replace "Release" with a product name ("React Shared Components" or maybe something made up).

Reply:
> > Some did, but I think that they all do now: [4916e57](https://github.com/sonatype/sonatype-react-shared-components/commit/4916e577a5ea2d0331ef672b9cc87fe76505d37c)
> > Make sure you look at the page for NxGlobalSidebarFooter.
> 
> This didn't address what I said. It is my understanding that, most typically, the text for this property will be the product name followed by the version number. In your examples you just have "Release" followed by the version number. I am suggesting that we make them more realistic by replacing "Release" with a product name ("React Shared Components" or maybe something made up).

I changed one of the examples here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/95b8f3c13b4f7dbd8d20a6033540c1f70e74ef95

But I'm not sure where you're getting the understanding that "most typically, the text for this property will be the product name followed by the version number" since the mockups just say "Product Release Name" and I can't recall any discussion on the subject (outside of this PR).